apps/taquito-test-dapp/src/lib/TestContainer.svelte

```diff
@@ -335,36 +335,12 @@
           {#if opHash}
             <div style="word-break:break-word;">
               Operation hash:
-              {#if $store.networkType === NetworkType.ITHACANET}
-                <a
-                  href={`https://better-call.dev/ithacanet/opg/${opHash}/contents`}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {shortenHash(opHash)}
-                </a>
-              {:else if $store.networkType === NetworkType.HANGZHOUNET}
-                <a
-                  href={`https://better-call.dev/hangzhounet/opg/${opHash}/contents`}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {shortenHash(opHash)}
-                </a>
-              {:else if $store.networkType === NetworkType.GHOSTNET}
-                <a
-                  href={`https://better-call.dev/ghostnet/opg/${opHash}/contents`}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {shortenHash(opHash)}
-                </a>
-              {:else if $store.networkType === NetworkType.MAINNET}
-                <a href={`https://tzkt.io/${opHash}`} target="_blank" rel="noopener noreferrer">
+              {#if $store.networkType === NetworkType.CUSTOM}
+                {shortenHash(opHash)}
+              {:else}
+                <a href={`${getTzKtUrl($store.networkType)}/${opHash}`} target="_blank" rel="noopener noreferrer">
```

Comment:
Curious of the reason to change from better-call.dev to tzkt

Reply:
AFAIK TzKt is the newer, more feature rick, more actively developed successor to BCD.